### DISCUSSION TITLE
Page references

### DISCUSSION THREAD

**Discussion by MinskLeo** [docmost/docmost#418] [Ideas]
Hi! First of all - great tool! I love the simplicity of the tool while still having great amount structure/styling possibilities. But the single, very essential thing missing - page references. This is what I was defenitelly expected from this kind of apps, to be able to reference one page from another and directly move between them without opening second tab, but unfortunatelly wasn't able to get how to do that. Most chances it's not implemented yet. That's why I'm here - please add this feature 🙏 🙏 🙏 .

Also there is another similar discussion https://github.com/docmost/docmost/discussions/283 but it was not answered.

**Comment by Philipinho** [comment]
There is an open issue https://github.com/docmost/docmost/issues/93.

**Reply by MinskLeo** [reply]
Thanks for answering! I'll be following discussion which you referenced, thanks.
